### DISCUSSION TITLE
SSL for snapshot images urls

### DISCUSSION THREAD

**Discussion by ntbritton** [mrlt8/docker-wyze-bridge#777] [General]
I use SSL with my install, and that works great for the video feed, but when I try to look at snapshot https://ha_ip:8887/snapshot/shed/, it fails, but works with just HTTP.    

Am I missing a config, or is that expected?    I am trying to expose the snapshots over sharptools, but need that to be https.

**Comment by mrlt8** [comment]
The WebUI is still pretty basic and doesn't support SSL yet.
